index.js

```diff
@@ -1,28 +1,19 @@
 'use strict';
 var url = require('url');
 var got = require('got');
-var registryUrl = require('registry-url');
-var rc = require('rc');
+var registryUrl = require('registry-auth-token/registry-url');
```

Comment:
I originally added the registry-url code to the auth token repo because a PR I sent to a different project ([ied](https://github.com/alexanderGugel/ied) wanted to reduce the number of dependencies they added. I'd be fine with using the `registry-url` module if you prefer the extra dependency.